examples/01_typescript/src/App.tsx

```diff
@@ -139,14 +139,13 @@ query($code: ID!) {
   }
 }
 `,
-  (get) => ({ code: get(codeAtom) }),
-  undefined,
-  () => client
-)
+  getVariables: (get) => ({ code: get(codeAtom) }),
+  getClient: () => client,
+})
```

Comment:
The reason we prefer function arguments to one object argument is to avoid property names for better minification. It's also how `jotai` is designed.
It's a subtle thing, so if you prefer an object argument, I don't mind.

Reply:
Object argument is so much better in this case due to multiple options that are optional. Surely it's possible to achieve with method overload, but that would make typings major pain to handle. So, yes, I would prefer options here to be closer to how it's done in original urql and similar to original Apollo bindings too.

Reply:
I'm fine and making is similar with urql react binding would be important.